test/lib/tpunit++.cpp

```diff
@@ -395,7 +395,7 @@ void tpunit::TestFixture::tpunit_detail_do_tests(TestFixture* f) {
 
           // Dump the test buffer if the test included any log lines.
           f->printTestBuffer();
-          printf("\xE2\x9D\x8C %s\n", t->_name);
+          printf("\xE2\x9D\x8C !FAILED! \xE2\x9D\x8C %s\n", t->_name);
```

Comment:
```suggestion
          printf("\xE2\x9D\x8CFAILED\xE2\x9D\x8C %s\n", t->_name);
```
NAB I think this would look better

Reply:
No, because then you can't easily search for it in the travis output (which for some reason contains at least 10 occurrences of "failed" in successful builds)

Reply:
OH sorry, you can have the last space 😓

Reply:
@tylerkaraszewski if you like the suggestion better, feel free to commit it and merge 🙃

Reply:
I'm fine with the `!!` I agree it's easier to search for. We do list the failed test names at the end of the output, but if this is helpful too then that's fine.